packages/@react-aria/listbox/docs/useListBox.mdx

```diff
@@ -133,10 +133,12 @@ function ListBox(props) {
   // Get props for the listbox element
   let ref = React.useRef();
   let {listBoxProps} = useListBox({...props, ref}, state);
+  let labelProps = useLabels(props);
```

Comment:
Good catch. I am working on an update to most of our react-aria hooks to pass through DOM props and aria labels as part of the hook itself, so this won't be necessary once that's done (will be built into useListBox).

Question: do you think we should build a visible label into ListBox as well, and show these examples with a visible label instead? `useListBox` could return props for the label similar to how it does for useSelect, useProgressBar, etc.

Reply:
If label works the way we handle it for other form controls we should probably do that with the console warning when no aria-label, aria-labelledby or label is provided. My only concern for adding label to listbox would be about layout. Are we flexible enough for it ever to be used if implemented.

Reply:
Yeah. I was thinking only of adding it to the `useListBox` hook and not React Spectrum `ListBox` for now since there's no design for it. It would just be a convenience for people making custom listbox components with a visual label. Plus, that hook already has the logic in it to warn when a visible label and aria label are not present. I'll handle that in my other PR.

Reply:
https://github.com/adobe-private/react-spectrum-v3/pull/534/files#diff-d37256b18094839bfce33ec0e1323ead